test/wrapper.test.js

```diff
@@ -300,7 +300,7 @@ test('creates pretty write stream with custom options for pino-pretty, via prett
   const dest = new Writable({
     objectMode: true,
     write (formatted, enc) {
-      t.is(formatted, 'INFO\t: foo\n')
+      t.is(formatted, 'INFO : foo\n')
```

Comment:
```suggestion
      t.is(formatted, 'INFO: foo\n')
```

Reply:
I get errors when trying to apply this change:

```console
$ git diff
diff --git a/test/wrapper.test.js b/test/wrapper.test.js
index 0ccb7be..eabca89 100644
--- a/test/wrapper.test.js
+++ b/test/wrapper.test.js
@@ -300,7 +300,7 @@ test('creates pretty write stream with custom options for pino-pretty, via prett
   const dest = new Writable({
     objectMode: true,
     write (formatted, enc) {
-      t.is(formatted, 'INFO : foo\n')
+      t.is(formatted, 'INFO: foo\n')
       t.done()
     }
   })
@@ -314,7 +314,7 @@ test('creates pretty write stream with custom options for pino-pretty, via opts
   const dest = new Writable({
     objectMode: true,
     write (formatted, enc) {
-      t.is(formatted, 'INFO : foo\n')
+      t.is(formatted, 'INFO: foo\n')
       t.done()
     }
   })

$ npm test

> pino-multi-stream@5.2.0 test /Users/mmarchini/workspace/misc/pino-multi-stream
> standard && tap --100 test/*.test.js

 PASS  test/multistream.test.js 50 OK 108.33ms
 PASS  test/bunyan-level.test.js 126 OK 169.296ms
 FAIL  test/wrapper.test.js
 ✖ should be equal

  test/wrapper.test.js                        
  301 |     objectMode: true,                 
  302 |     write (formatted, enc) {          
> 303 |       t.is(formatted, 'INFO: foo\n')  
      | --------^                             
  304 |       t.done()                        
  305 |     }                                 
  306 |   })                                  

  --- expected    
  +++ actual      
  @@ -1,2 +1,2 @@ 
  -INFO: foo      
  +INFO : foo     

  test: test/wrapper.test.js creates pretty write stream with custom options for
    pino-pretty, via prettyPrint (default)
  stack: |
    Writable.write [as _write] (test/wrapper.test.js:303:9)
    Object.write (node_modules/pino/lib/tools.js:283:12)
    Object.write (multistream.js:68:18)
    Pino.write (node_modules/pino/lib/proto.js:177:15)
    Pino.LOG [as info] (node_modules/pino/lib/tools.js:57:21)
    Test.<anonymous> (test/wrapper.test.js:310:7)
    Object.<anonymous> (test/wrapper.test.js:299:1)
    Module.replacementCompile (node_modules/append-transform/index.js:58:13)
    Object.<anonymous> (node_modules/append-transform/index.js:62:4)

 PASS  test/level.test.js 78 OK 162.055ms
 FAIL  test/wrapper.test.js
 ✖ should be equal

  test/wrapper.test.js                        
  315 |     objectMode: true,                 
  316 |     write (formatted, enc) {          
> 317 |       t.is(formatted, 'INFO: foo\n')  
      | --------^                             
  318 |       t.done()                        
  319 |     }                                 
  320 |   })                                  

  --- expected    
  +++ actual      
  @@ -1,2 +1,2 @@ 
  -INFO: foo      
  +INFO : foo     

  test: test/wrapper.test.js creates pretty write stream with custom options for
    pino-pretty, via opts (obsolete)
  stack: |
    Writable.write [as _write] (test/wrapper.test.js:317:9)
    Object.write (node_modules/pino/lib/tools.js:283:12)
    Object.write (multistream.js:68:18)
    Pino.write (node_modules/pino/lib/proto.js:177:15)
    Pino.LOG [as info] (node_modules/pino/lib/tools.js:57:21)
    Test.<anonymous> (test/wrapper.test.js:324:7)
    Object.<anonymous> (test/wrapper.test.js:313:1)
    Module.replacementCompile (node_modules/append-transform/index.js:58:13)
    Object.<anonymous> (node_modules/append-transform/index.js:62:4)

 FAIL  test/wrapper.test.js 2 failed of 53 158.465ms
 ✖ should be equal
 ✖ should be equal


                         
  🌈 SUMMARY RESULTS 🌈  
                         

 FAIL  test/wrapper.test.js 2 failed of 53 158.465ms
 ✖ should be equal
 ✖ should be equal

Suites:   1 failed, 3 passed, 4 of 4 completed
Asserts:  2 failed, 305 passed, of 307
Time:     1s
----------------|----------|----------|----------|----------|-------------------|
File            |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------------|----------|----------|----------|----------|-------------------|
All files       |      100 |      100 |      100 |      100 |                   |
 index.js       |      100 |      100 |      100 |      100 |                   |
 multistream.js |      100 |      100 |      100 |      100 |                   |
----------------|----------|----------|----------|----------|-------------------|

```

Reply:
Oh wait, I was on an older version of `pino-pretty` (because I was trying to find out where things broke). Lemme commit with those changes.